nodejs/README.md

```diff
@@ -0,0 +1,208 @@
+ES5, but when writing code on the server, you don't have to write
+browser-compatible code. With `--harmony`, Node.js [supports many ES6 and ES7
+features](http://node.green/).
+
+You can also use the same build tools that you use on the front end, like
+webpack and browserify with babel, to transpile your code. This allows you to
+fill in the feature gaps that are not yet supported by Node.js. This also means
+your browser code and server code use the same ECMAScript features.
+
+
+### Best practices
+
+
+#### Recommend
+
+- Use the [current active LTS](https://github.com/nodejs/LTS#lts-schedule1)
+  version of Node.js.
+
+
+#### Consider
+
+- What syntax are you using on the front end? Consider using the same syntax and
+  features for both the  front end and back end code.
+- Many developers write JavaScript in the browser (ES5) even if they aren't
+  a JavaScript developer. Using a syntax that many folks are familiar with
+  allows more folks to jump in.
+- Front end build tools make it easy to choose a syntax you want and polyfill
+  the missing features in the browser.
+
+
+## Frameworks
+
+Frameworks provide a common baseline for your application. ATO requires
+configurations that most frameworks provide out of the box, or make it very easy
+to enable.
+
+The most common framework at 18F is [Express][expressjs]. You don't have to
+choose Express, but you should use an existing framework with an active
+community for support.
+
+Here are some existing projects at 18F using Express:
+
+- [Federalist][github-federalist]
+- [OMB eRegs][github-omb-eregs]
+- [Analytics Reporter API][github-analytics-report]
+
+You can [find many
+more](https://github.com/search?q=org%3A18F+filename%3Apackage.json+express&type=Code&utf8=%E2%9C%93) internal and
+external projects as well.
+
+
+### Best practices
+
+
+#### Recommend
+
+- Use an existing framework with active community support.
+
+
+#### Suggest
+
+- Use [Express][expressjs].
+
+
+## Scaffolding your application
+
+Scaffolding tools can help take care of some of boiler plate of initializing
+a project. Some scaffolds are generator tools, others are kits intended to be
+forked. There are many available on Github. You might also choose one based on the
+front end frameworks you want to use and then add the server-side pieces
+yourself. You also don't have to use one at all.
+
+Here's a list of scaffolds we've had success with:
+
+- [express-generator](https://www.npmjs.com/package/express-generator)
+- [express-babel](https://github.com/vmasto/express-babel)
+
+
+### Best practices
+
+
+#### Consider
+
+- What front end frameworks do you plan to use? Maybe choose a generator based
+  on those frameworks instead.
+- Don't spend too much time finding the perfect generator. Once your application
+  has been scaffolded, you're not going to scaffold it again.
+
+
+## Authentication
+
+For Express, [Passport](http://www.passportjs.org/) has many authentication
```

Comment:
This sentences seems like it belongs down in the recommendation portion of this section. Other sections of the guide give a brief overview of what the section is about.

To that end, maybe title this section "Authentication and Authorization" and provide concise definitions for differentiating the two as the introductory text here.